code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -106,10 +106,10 @@
 	name = "Fireball"
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
 
+/*
 /datum/spellbook_entry/rod_form
 	name = "Rod Form"
-	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form
-	cost = 40
+	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form */
```

Comment:
It can be admin-given, rather than purchased in spellbooks entirely. 

For real though, it's not even a good idea to begin with. Yes let's become an object that instantly kills everyone it touches and punches through R-wall without issue.